dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -388,6 +389,58 @@ public Workflow refresh(@ApiParam(hidden = true) @Auth User user,
         return workflow;
     }
 
+    @GET
+    @Path("/upsertVersion/")
```

Comment:
I know we haven't really followed REST much.  But if we were going to, i'd somehow avoid putting something after /workflow that isn't {workflowId}

Reply:
Any suggestions?

Reply:
@coverbeck any ideas?

Reply:
@coverbeck does PUT make more sense?